eos/form-factors/analytic-b-to-pi-pi.cc

```diff
@@ -528,6 +529,145 @@ namespace eos
         return std::imag(this->f_time(q2, k2, z));
     }
 
+    std::array<complex<double>, 3>
+    AnalyticFormFactorBToPiPiBFvD2016::f_perp(const double & q2, const double & k2) const
+    {
+        std::array<complex<double>, 3> res;
+
+        std::function<complex<double>(const double &)> integrandP = [&] (const double & x)
+        {
+            return 0.5 / std::sqrt(3.0) * this->f_perp(q2, k2, x);
+        };
+
+        std::function<complex<double>(const double &)> integrandD = [&] (const double & x)
+        {
+            return 0.5 / std::sqrt(5.0) * x * this->f_perp(q2, k2, x);
+        };
+
+        unsigned npoints = 1000;
```

Comment:
I am pretty sure that ``integrate1D`` requires a power of 2 as the number of integration points.
Also, ``integrate1D`` is deprecated. Can you allocate a suitable workspace upon construction of the observable and use e.g. ``integrate<gsl::QNG>``?